packages/blade/src/components/Box/BaseBox/getBaseBoxStyles.web.test.ts

```diff
@@ -145,26 +93,18 @@ describe('getBaseBoxStyles', () => {
     });
     const boxStylesWithoutUndefined = JSON.parse(JSON.stringify(boxStyles));
     expect(boxStylesWithoutUndefined).toMatchInlineSnapshot(`
-      Object {
-        "@media screen and (min-width: 1200px)": Object {
-          "margin": "auto",
-        },
-        "@media screen and (min-width: 480px)": Object {
-          "margin": "2px 12px 100%",
-        },
-        "@media screen and (min-width: 768px)": Object {
-          "margin": "22px",
-        },
-        "margin": "2px",
-      }
-    `);
-  });
-
-  it('should add no css', () => {
-    const boxStyles = getBaseBoxStyles({
-      theme: paymentLightTheme,
-    });
-    const boxStylesWithoutUndefined = JSON.parse(JSON.stringify(boxStyles));
-    expect(boxStylesWithoutUndefined).toMatchInlineSnapshot(`Object {}`);
+    Object {
+      "@media screen and (min-width: 1200px)": Object {
+        "margin": "auto",
+      },
+      "@media screen and (min-width: 480px)": Object {
+        "margin": "2px 12px 100%",
+      },
+      "@media screen and (min-width: 768px)": Object {
+        "margin": "22px",
+      },
+      "margin": "2px",
+    }
+  `);
```

Comment:
Minor: can we add another explicit test for asserting media queries aren't added for non responsive styles?

Reply:
Yup its here - 

https://github.com/razorpay/blade/blob/e3b1ef0a07ee63c9aed5525d2027e0c21b0e6b37/packages/blade/src/components/Box/BaseBox/getBaseBoxStyles.test.ts#L52-L60

I have kept common tests for web and native in `getBaseBoxStyles.test.ts`